mixer/cmd/config.go

```diff
@@ -96,11 +98,49 @@ var configSetCmd = &cobra.Command{
 	},
 }
 
+var configGetCmd = &cobra.Command{
+	Use:   "get <property>",
+	Short: "Get a property in the config file",
+	Long: `This command will parse the provided property in the format 'Section.Property'
+	and return its value. The value returned is the value used by mixer, which will be
+	either the value set in the config file or the default value if the property has not
+	been defined in the file.`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var value string
+
+		// Look for the value in mixer.state first
+		var ms config.MixState
+		err := ms.Load("")
+		if err == nil {
+			value, err = config.GetProperty(&ms, args[0])
+			if err == nil {
+				fmt.Println(value)
```

Comment:
It'd be useful to know this value came from` [mixer.state]:` or `[builder.conf]:`

Reply:
I believe devops want to use the output of this command for automation, which is much simpler to do if it just returns the value itself.
But I agree that for using it manually it is nice to see where it came from.

Reply:
@George can you confirm this?

Reply:
I think you've got the wrong @george 😉

Reply:
@tmarcu, I would assume that the point of the CLI is for DevOps shops to not need to know where the values come from?  I see it as an implementation behind the hood.